pegasus/sites.v3/code.org/public/ai/index.haml

```diff
@@ -60,15 +53,7 @@ theme: responsive_full_width
             %p.no-margin-bottom=hoc_s(:ai_intro_list_desc_03)
     .clear
 
-%section#ai-pl-callout.bg-primary-light.blog-post-callout
-  .wrapper
-    %i{class: "#{icon_megaphone}"}
-    .text-wrapper
-      %h2.heading-lg=hoc_s(:ai_pl_callout_title)
-      %p
-        %strong=hoc_s(:ai_pl_callout_desc_02)
-      %p.body-three=hoc_s(:ai_pl_callout_desc_01)
-      %a.link-button{href: "/ai/pl/101"}=hoc_s(:signup_submit_explore)
+= view :dance_party_ai_skinny_banner
```

Comment:
Added the new banner here, and removed the AI 101 callout (`#ai-pl-callout`)

<img width="800" alt="AI page" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/147528ad-2577-4ecd-a48f-6fd0450ebd6f">